.eslintrc.json

```diff
@@ -24,6 +24,7 @@
 	"rules": {
 		"@typescript-eslint/no-use-before-define": "off",
 		"comma-dangle": "off",
+		"i18n-text/no-en": "off",
```

Comment:
Not sure how this works but I need to research this i18n rules

Reply:
that was another eslint rule.
i had updated the dependencies for the repo when I installed the GitHub GraphQL API lib and that's what caused all of these other issues.
I would prefer to have those string internationalized, but I also don't know how to go about doing that at this point, so I just turned the rule off.